book/src/custom.md

```diff
@@ -0,0 +1,37 @@
+# Custom statements and custom operations
+
+<font color="red">TODO</font>: Settle on a design to protect against unwanted custom operations in upstream PODS.  One such design option is proposed in <font color="green">green</font>.
+
+Users of the POD system can introduce _custom statements_ to express complex logical relations not available in the built-in statements.  Custom statements are built on top of existing statements (built-in or custom) using logical _custom operations_.
+
+Every MainPod has room to introduce a number of _custom statements_ and _custom operations_. <font color="green"> As a security requirement, every custom operation introduced in a MainPod must output a custom statement introduced in the same MainPod. </font> When a custom statement is introduced in a MainPod, it becomes available for use in that POD and all PODs that inherit[^inherit] from it.
+
+## Custom statements and IDs
+
+<font color="green"> The public data in every MainPod includes a list of all custom statements and custom operations used in that POD and all ancestor PODs. </font>
+
+A custom statement, like a built-in statement, is identified by a _name_ on the front end and an _identifier_ on the back end.  Both name and identifier must be unique -- that is, distinct from the names and identifiers of all custom statements in this POD and all ancestor PODs.  <font color="green"> The POD circuit verifies that the identifier is unique. </font>
```

Comment:
`Both name and identifier must be unique` how do we enforce it?, like, how to prevent conflicts/collusions between different custom statement identifiers between a POD and its ancestors